### ISSUE TITLE
Verb scores are always the same

### ISSUE THREAD

**Issue by yuzheyao22** [enlighten0707/Body-Part-Map-for-Interactiveness#8]
If I print the "vs"(verb_scores) in hoi.py, logits for all the 117 verbs are the same. It means that the model does not predict the verb, right?

**Comment by yuzheyao22** [comment]
when printing the outputs['pred_verb_logits']`torch.Size([4, 64, 117])tensor([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       device='cuda:2')`

**Comment by yuzheyao22** [comment]
> If I print the "vs"(verb_scores) in hoi.py, logits for all the 117 verbs are the same. It means that the model does not predict the verb, right?

For example when printing the **_outputs['pred_verb_logits'][0][0]_**
`tensor([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
        0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       device='cuda:2')`

**Comment by enlighten0707** [comment]
Yes, the provided code output interactiveness predictions, which are used to apply non-interaction suppression (NIS) on verb classification results. Here "pred_verb_logits" is omitted.
